frontend/public/components/marketplace/kubernetes-marketplace-items.jsx

```diff
@@ -208,7 +208,7 @@ export class MarketplaceTileViewPage extends React.Component {
       <CatalogTileView.Category totalItems={items.length} viewAll={true}>
         {_.map(items, item => {
           const { obj, name, imgUrl, iconClass, provider, description } = item;
-          const uid = obj.metadata.uid;
+          const uid = `${obj.metadata.name}/${obj.metadata.labels['catalog-namespace']}`;
```

Comment:
Should use the `status.catalogSourceNamespace` instead of the label: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/package-server/apis/packagemanifest/v1alpha1/packagemanifest_types.go